ios/Codeforces Watcher/Application/features/contests/ContestsViewController.swift

```diff
@@ -183,4 +184,81 @@ class ContestsViewController: UIHostingController<ContestsView>, ReKampStoreSubs
     private func fetchContests() {
         store.dispatch(action: ContestsRequests.FetchContests(isInitiatedByUser: true))
     }
+    
+    private func updateFilters(_ state: ContestsState) {
+        let filters = state.filters
+
+        rootView.filterItems = [
```

Comment:
Aren't all items very similar here? Maybe you could add some method that builds the filter item for you by taking `title` and `platform` enum.